src/status_im/ui/screens/chats_list/views.cljs

```diff
@@ -92,7 +92,7 @@
                                                              [common/list-footer]
                                                              [common/bottom-shadow]]))
                        :renderSeparator renderers/list-separator-renderer
-                       :style           (st/list-container tabs-hidden?)}]
+                       :style           (st/list-container)}]
```

Comment:
is `st/list-container` a function?

Reply:
It's a defnstyle macro https://github.com/status-im/status-react/pull/1916/files/611772f62cff8b959c1eabf39fe5f7c21c3b6647#diff-c4ed90e70a6bbfc75938d8434b09d5eeR143

Reply:
you can change it on `defstyle` if there are no params